src/my-view1.html

```diff
@@ -246,6 +248,16 @@ <h2 class="section-title bg-secondary center">Select a location</h2>
       <section id="preview">
         <ryr-email-templatizer templatevars="{{apivars}}" htmlrenderedtemplate="{{buffer}}" markdownrenderedtemplate="{{md}}"></ryr-email-templatizer>
         <h2 class="section-title bg-primary center">Preview</h2>
+        
+        <div class="warning-box">
+          <iron-icon icon="vaadin:warning"></iron-icon>
+          <p class="justify">
+            Make sure the business you chose supports your request. Without it, the request will be not be processed. You must also go talk to the businesses supporting their request in order for the bike rack to be installed.
+            <br>
+            <strong>There is no guarantee that the bike rack will be installed.</strong> 
```

Comment:
I like this very much! Please check with @2echoi and @UX-Chris to see how we can make it fit the new design.